pkg/kubectl/pdb_test.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Copyright 2016 The Kubernetes Authors.
->
Copyright 2017 The Kubernetes Authors.
This is file creation time.

Reply:
@zhangxiaoyu-zidif addressed